pgsql/pc_access.c

```diff
@@ -860,6 +864,46 @@ Datum pc_version(PG_FUNCTION_ARGS)
 	PG_RETURN_TEXT_P(version_text);
 }
 
+PG_FUNCTION_INFO_V1(pc_pgsql_version);
+Datum pc_pgsql_version(PG_FUNCTION_ARGS)
+{
+	text *version_text;
+	char version[12];
+	snprintf(version, 12, "%d", PGSQL_VERSION);
+	version_text = cstring_to_text(version);
+	PG_RETURN_TEXT_P(version_text);
+}
+
+PG_FUNCTION_INFO_V1(pc_libxml2_version);
+Datum pc_libxml2_version(PG_FUNCTION_ARGS)
+{
+	text *version_text;
+	char version[64];
+	snprintf(version, 64, "%s", LIBXML2_VERSION);
```

Comment:
Is it wanted to have an integer formatting in case of the pgsql version and a string formatting for libxml?

Reply:
Yes, the pgsql version has this form: `96`. While the libxml2 version has this form: `2.9.4`.